gxupload_0

```diff
@@ -0,0 +1,2 @@
+sample data
```

Comment:
Pretty sure you didn't want to add this.

Reply:
Definitely not, thanks!